src/site/paragraphs/table.drupal.liquid

```diff
@@ -27,31 +27,38 @@
     }
 {% endcomment %}
 
-
-<table data-template="paragraphs/table" data-entity-id="{{ entity.entityId }}" class="va-table">
-
-    {% if entity.fieldTable.caption %}
-        <caption>{{ entity.fieldTable.caption }}</caption>
-    {% endif %}
-
-    {% for value in entity.fieldTable.value %}
-        {% if forloop.first == true %}
-            <thead>
-                <tr>
-                    {% for column in value %}
-                        <th>{{ column }}</th>
-                    {% endfor %}
-                </tr>
-            </thead>
-        {% elsif forloop.last == false %}
-            <tbody>
-                <tr class="class">
+<table data-template="paragraphs/table" data-entity-id="{{ entity.entityId }}" class="va-table va-table--responsive" role="table">
+    <tbody>
+        {% if entity.fieldTable.caption %}
+            <caption>{{ entity.fieldTable.caption }}</caption>
+        {% endif %}
+        {% assign colLabels = entity.fieldTable.value.0  %}
+        {% for value in entity.fieldTable.value %}
+            {% if forloop.first == true %}
+                <thead>
+                    <tr role="row">
+                        {% for column in value %}
+                            <th role="columnheader" scope="col">{{ column }}</th>
+                        {% endfor %}
+                    </tr>
+                </thead>
+            <!-- this elsif is here to prevent the caption from showing up as a row -->
+            {% elsif forloop.last == false %}
+                <tr class="vads-u-padding-top--5" role="row">
                     {% for column in value %}
-                        <td>{{ column }}</td>
+                        <dfn>
```

Comment:
I think rendering the `dfn` as an immediate child of `tr` is technically invalid HTML. 

>**4.9.8 The `tr` element - Content Model**: Zero or more `td`, `th`, and script-supporting elements. - https://html.spec.whatwg.org/multipage/tables.html#the-tr-element

I think it would be valid if you moved the `dfn` to be a child of the `td` instead of being a child of the `tr`.

Reply:
done